src/build_scripts/build_stdlib.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -eu
+
+## preconditions
+if [ -z "${GERBIL_SETUP+x}" ]; then
+  (1>&2 echo "ERROR: Do not call this script directly.") && exit 1
```

Comment:
same here, I call the script directly all the time when I do stdlib development.